docs/src/index.md

```diff
@@ -0,0 +1,118 @@
+# GeoMakie.jl
+GeoMakie.jl is a Julia package for plotting geospatial data on a given map projection. It is based on the [Makie.jl plotting ecosystem](https://makie.juliaplots.org/stable/).
+
+The package [ClimateBase.jl](https://juliaclimate.github.io/ClimateBase.jl/dev/) builds upon GeoMakie.jl to create a seamless workflow between analyzing/manipulating climate data, and plotting them.
+
+## Installation
+
+This package is **in development** and may **break**, although we are currently working on a long-term stable interface. 
+
+You can install it from the REPL like so:
+```julia
+]add GeoMakie
+```
+
+## API
+Using GeoMakie.jl is straightforward, although it does assume basic knowledge of the Makie.jl ecosystem. 
+
+GeoMakie.jl provides an axis for plotting geospatial data, [`GeoAxis`](@ref), and also the function [`geo2basic`](@ref) that converts an output of GeoJSON to a polygon appropriate for plotting. Both are showcased in the examples below.
+
+```@docs
+GeoAxis
+geo2basic
+```
+
+
+## Examples
+
+### Surface example
+```@example MAIN
+using GeoMakie, CairoMakie
+
+lons = -180:180
+lats = -90:90
+field = [exp(cosd(l)) + 3(y/90) for l in lons, y in lats]
+
+fig = Figure()
+ax = GeoAxis(fig[1,1])
+el = surface!(ax, lons, lats, field; shading = false)
+fig
+```
+
+### Scatter example
+```@example MAIN
+using GeoMakie, CairoMakie
+
+lons = -180:180
+lats = -90:90
+slons = rand(lons, 2000)
+slats = rand(lats, 2000)
+sfield = [exp(cosd(l)) + 3(y/90) for (l,y) in zip(slons, slats)]
+
+fig = Figure()
+ax = GeoAxis(fig[1,1])
+el = scatter!(slons, slats; color = sfield)
+fig
+```
+
+### Different map projection
```

Comment:
```suggestion
### Map projections
The default projection is given by the arguments `source = "+proj=longlat +datum=WGS84", dest = "+proj=eqearth"`, so that if a different one is needed, for example a `wintri` projection one can do it as follows:
```